src/mpl/include/mpl_gavl.h

```diff
@@ -34,8 +35,11 @@ enum {
 };
 
 enum {
+    /* range search */
     MPLI_GAVL_SUBSET_SEARCH,
-    MPLI_GAVL_INTERSECTION_SEARCH
+    MPLI_GAVL_INTERSECTION_SEARCH,
+    /* address search */
+    MPLI_GAVL_ADDR_SEARCH
```

Comment:
How about `MPLI_GAVL_START_ADDR_SEARCH` to be more precise? Otherwise, I intuitively thought any address within the range can be searched.

Reply:
fixed